lib/emcbUDPconstants.js

```diff
@@ -5,7 +5,7 @@ var constants = {
 
     // Application Layer Constraints
     EMCB_UDP_IMPLEMENTED_PROTOCOL_VERSION                        : 1,
-    EMCB_UDP_MESSAGE_THROTTLE_TIME_MS                            : 200,
+    EMCB_UDP_MESSAGE_THROTTLE_TIME_MS                            : 1000,
```

Comment:
Is there a reason why we increased the delay for throttling the messages?  Is this to allow for open and close actions?

Reply:
I think that was the reason. For the EV patch settings endpoint I had to bump this up all the way to 5000 because switching mode can take a few seconds. We really need per-message timeouts, but I think that would be a rather significant change.